server/routes/api/documents.ts

```diff
@@ -1063,6 +1064,64 @@ router.post("documents.delete", auth({ member: true }), async (ctx) => {
   };
 });
 
+router.post("documents.empty_trash", auth({ admin: true }), async (ctx) => {
+  const { user } = ctx.state;
+
+  const deleted: { documentId: string; collectionId: string }[] = [];
+
+  await Document.unscoped().findAllInBatches<Document>(
+    {
+      where: {
+        teamId: user.teamId,
+        deletedAt: {
+          [Op.ne]: null,
+        },
+      },
+      paranoid: false,
+      limit: DocumentValidation.emptyTrash,
+    },
+    async (documents) => {
```

Comment:
I think there's a opportunity to speed this up with `Promise.all` after a filter on `authorize` and `documentPermanentDeleter(documents)`